### DISCUSSION TITLE
35

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#802] [Comments]
http://jojoldu.tistory.com/35

**Comment by jojoldu** [comment]
작성자: soultomind

동욱님 잘 보고 갑니다 항상 열심히 하시는거 같아 부럽습니다 ㅎ

**Comment by jojoldu** [comment]
작성자: 창천향로

앗 바우님! 오랫만이에요!! ㅎㅎ 잘지내시죠?? 전 아직 멀었습니다 ㅠㅠ ㅎㅎ;; 스터디때 바우님이 정말 열심히 하시던게 생각나네요! ㅎㅎ

**Comment by gonghojin** [comment]
오늘도 잘 읽고 갑니다~

**Comment by jojoldu** [comment]
@gonghojin 고맙습니다 :)
